pkg/cephmgr/mon/config.go

```diff
@@ -303,6 +304,7 @@ func CreateDefaultCephConfig(context *clusterd.Context, cluster *ClusterInfo, ru
 			OsdObjectStore:         store,
 			RbdDefaultFeatures:     3,
 			CrushtoolPath:          "",
+			FatalSignalHandlers:    "false",
```

Comment:
yes we lose the recent events functionality. I can't decide if its helpful or not. we can bring it back if we really miss it.